tests/phpunit/admin/migrations/class-wc-stripe-subscriptions-repairer-legacy-sepa-tokens-test.php

```
@@ -89,7 +89,7 @@ public function tear_down() {
 	 * We can't mock the check for WC_Subscriptions, so we'll test the rest of the conditions.
 	 */
 	public function test_updater_gets_scheduled_on_right_conditions() {
-		WC_Stripe_Helper::update_main_stripe_settings( [ 'upe_checkout_experience_enabled' => 'yes' ] );
+		WC_Stripe::write_settings_option( [ 'upe_checkout_experience_enabled' => 'yes' ] );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe' in method 'test_updater_gets_scheduled_on_right_conditions'. (undefined)

(StaticAccess)